clients/src/main/java/org/apache/kafka/clients/consumer/internals/StreamsMembershipManager.java

```diff
@@ -249,12 +250,12 @@ private void transitionToJoining() {
     private void transitionToSendingLeaveGroup(boolean dueToExpiredPollTimer) {
         if (state == MemberState.FATAL) {
             log.warn("Member {} with epoch {} won't send leave group request because it is in " +
-                "FATAL state", memberIdInfoForLog(), memberEpoch);
+                "FATAL state", memberId, memberEpoch);
             return;
```

Comment:
Removed `memberIdInfoForLog()` method since we know the member id from the start, here and below